CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ X.Y.Z Release notes
 * Fixed a bug in 3rd party JSON parser: `localeconv()` does not exist on Android API < 21 and should not be called.
 
 ### Internal
-* None.
+* Upgraded to Realm Sync 2.1.8.
```

Comment:
Anything from that release that should be mentioned?

Reply:
No, it is primarily for internal projects. Should I remove the line?

Reply:
No that's perfect then. I've update it a bit